test/steps/run_steps.go

```diff
@@ -13,8 +13,9 @@ import (
 )
 
 // RunSteps defines Gherkin step implementations around running things in subshells.
+// nolint:funlen
 func RunSteps(suite *godog.Suite, fs *FeatureState) {
-	suite.Step(`^I run "([^"]+)"$`, func(command string) error {
+	suite.Step(`^I run "(.+)"$`, func(command string) error {
```

Comment:
This is required for embedded double in the string in https://github.com/git-town/git-town/blob/master/features/git-town-ship/current_branch/on_feature_branch/without_open_changes/commit_message_containing_double_quotes.feature#L14